distributed/tests/test_chaos.py

```diff
@@ -4,10 +4,17 @@
 
 from distributed import Nanny
 from distributed.chaos import KillWorker
-from distributed.utils_test import gen_cluster
+from distributed.utils_test import WINDOWS, gen_cluster
 
 
-@pytest.mark.parametrize("mode", ["sys.exit", "graceful", "segfault"])
+@pytest.mark.parametrize(
+    "mode",
+    [
+        "sys.exit",
+        "graceful",
+        pytest.param("segfault", marks=pytest.mark.skipif(WINDOWS, reason="unknown")),
```

Comment:
Does windows... not segfault? Weird.

Reply:
Yeah, not sure.

Reply:
Or the ctypes trick doesn't operate the same on windows and would need to find some other better trick.